src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps/OpHelper.cpp

```diff
@@ -402,8 +473,8 @@ AffineMapAttr getTiling2DTo4DMap(OpBuilder &b, Value val) {
   return AffineMapAttr::get(map);
 }
 
-AffineMapAttr getTiling3DTo4DMap(OpBuilder &b, Value val) {
-  assert(isTiling3DTo4D(val) &&
+AffineMapAttr getLeftmostTiling3DTo4DMap(OpBuilder &b, Value val) {
```

Comment:
Many of the prior operations where not specific if they applied to the right most or leftmost position, as only one was needed. As I added more, I made all names more explicit,